CODEOWNERS

```diff
@@ -31,3 +31,7 @@ instrumentation/racecar/ @chrisholmes @open-telemetry/ruby-contrib-maintainers @
 instrumentation/rspec/ @chrisholmes @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
 
 instrumentation/que/ @indrekj @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
+
+helpers/mysql/ @kaylareopelle @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
```

Comment:
he he... so nice you are adding your name twice?

Reply:
lol, yeah, I'm not sure if that makes sense! I'm happy to just close this, but it was a thought in my mind since the lambda PR requested a codeowner even though they're also part of the maintainers/approvers.

Reply:
Good point. I am not sure how to disambiguate the "owner" vs the reviewers.

Maybe we should follow the pattern in `otelcol-contrib` where the Readme files have an auto generated section:


> | Status        |           |
> | ------------- |-----------|
> | Stability     | [beta]: traces, metrics, logs   |
> | Distributions | [contrib], [observiq], [splunk], [sumo] |
> | Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aprocessor%2Fgroupbyattrs%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aprocessor%2Fgroupbyattrs) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aprocessor%2Fgroupbyattrs%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aprocessor%2Fgroupbyattrs) |
> | [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [`@rnishtala-sumo`](https://www.github.com/rnishtala-sumo) |

Reply:
If you're up for it sure!